payments/admin.py

```diff
@@ -62,8 +62,7 @@ def live(self, obj):
     live.short_description = 'Visible'
 
     def seats_available(self, obj):
```

Comment:
Can you relabel `seats_available` to `is_open` here to L65? That might be more clear for maintenance.